src/sys/resource.rs

```diff
@@ -0,0 +1,49 @@
+use std::mem;
+
+use libc::{self, c_int};
+pub use libc::{rlimit, RLIM_INFINITY};
```

Comment:
I *think* these should be reexported, but I'm not sure if we are consistently doing that.